src/exporter/models/exporterStore.ts

```diff
@@ -57,11 +58,14 @@ export const exporterStore = types
       const snapshot = getSnapshot(self.searchParams);
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const params: Record<string, unknown> = {};
+      // Get the source layer name
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      const sourceLayer: string = EXPORTER_CONFIG.ACTIVE_LAYER_PROPERTIES.urlPatternParams.layers as string;
```

Comment:
where are the changes in the config file? confd?

Reply:
no changes needed for confd , EXPORTER_CONFIG.ACTIVE_LAYER_PROPERTIES.urlPatternParams.layers is already the layer name config value